conf/nginx/via/direct_proxy.conf

```diff
@@ -0,0 +1,21 @@
+set $upstream $proxied_uri$is_args$args;
+proxy_ssl_server_name on;
+proxy_pass $upstream;
+proxy_redirect ~^(.*)$ $original_scheme://$http_host/proxy/static/$1;
+
+# Strip hypothesis cookies and authorization header.
+set $stripped_cookie $http_cookie;
+
+if ($stripped_cookie ~ "(.*)\s*auth=[^;]+;?(.*)") {
+    set $stripped_cookie $1$2;
+}
+if ($stripped_cookie ~ "(.*)\s*session=[^;]+;?(.*)") {
+    set $stripped_cookie $1$2;
+}
+proxy_set_header Cookie $stripped_cookie;
+proxy_set_header Authorization "";
+
+# Do not allow the third party server to set cookies.
+add_header "Set-Cookie" "";
+proxy_hide_header "Access-Control-Allow-Origin";
+add_header "Access-Control-Allow-Origin" $access_control_allow_origin;
```

Comment:
So this is just all the proxying code that we already had before, minus adding the caching headers, just extracted into a separate file so it can be reused via `include`

Reply:
Yep